sbncode/NuMuSelection/PandoraTesting_module.cc

```diff
@@ -0,0 +1,365 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       PandoraTesting
+// Plugin Type: analyzer (art v2_08_04)
+// File:        PandoraTesting_module.cc
+//
+// Generated at Mon Dec 18 08:00:14 2017 by Rhiannon Jones using cetskelgen
+// from cetlib version v3_01_01.
+////////////////////////////////////////////////////////////////////////
+
+// Playing around with Pandoraw objects
+// Taken from Rhiannon Jones github
```

Comment:
I think I found it, but there's no license. Could you inquire about copying/attribution?

Reply:
This wasn't being used in the analysis, so I removed it.